library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -277,8 +289,8 @@ public void onServiceDisconnected(ComponentName name) {
                 downloadsFilePersistence,
                 downloadsPersistence,
                 callbackThrottleCreator,
-                connectionChecker
-        );
+                connectionChecker,
+                downloadBatchRequirementRules);
```

Comment:
`);` on a newline to keep the formatting the same.

Reply:
sure